SECURITY.md

```diff
@@ -0,0 +1,27 @@
+# Security Policies and Procedures
+
+This document outlines security procedures and general policies for the `OnDemand`
+project.
+
+## Reporting a Bug
+
+If you have security concerns or think you have found a vulnerability in Open OnDemand,
+please contact us directly via email on the news list found [here](https://lists.osu.edu/mailman/listinfo/ood-users).
+
+Your email will be answered promptly by one of our developers.
+
+## Disclosure Policy
+
+When the security team receives a security bug report, they will assign it to a
+primary handler. This person will coordinate the fix and release process,
+involving the following steps:
+
+  * Confirm the problem and determine the affected versions.
+  * Audit code to find any potential similar problems.
+  * Prepare fixes for all releases still under maintenance. These fixes will be
+    released as fast as possible.
+
+## Comments on this Policy
+
+If you have suggestions on how this process could be improved please submit a
+pull request.
```

Comment:
I would say suggestions can come in any way, including,
* discourse topic
* an issue on this repo
* email in the ood-users list above
* Pull requests